Walkthroughs/ImageWhiteout/README.md

```diff
@@ -0,0 +1,215 @@
+ => [3/5] RUN apt update && apt install -y nodejs                                                                                                    9.5s
+ => [4/5] COPY MySecretPassword .                                                                                                                    0.0s
+ => [5/5] RUN rm -f MySecretPassword                                                                                                                 0.2s
+ => exporting to image                                                                                                                               0.2s
+ => => exporting layers                                                                                                                              0.2s
+ => => writing image sha256:95d8ed919c72b400e96ba2765797e12ae76cef6094f2b34b1d69a1a9be247bac                                                         0.0s
+ => => naming to docker.io/library/password-test                                                                                                     0.0s
+
+View build details: docker-desktop://dashboard/build/desktop-linux/desktop-linux/0rs3c1htzxrti3znimng1fmn6
+```
+
+#### Inspect the Image
+Let's use the `docker inspect` command to view the contents of our image; note that I am using a `jq` filter to cut 
+down the JSON being returned to just the part we are concerned with. This shows us the actual files on the disk of the 
+system running the docker daaemon.
+
+```shell
+$ docker inspect password-test| jq '.[].GraphDriver'
+{
+  "Data": {
+    "LowerDir": "/var/lib/docker/overlay2/6fi0uii6ovnuhv9ovglexzb23/diff:/var/lib/docker/overlay2/p5rypolie52umbkz04kncs6df/diff:/var/lib/docker/overlay2/lyumsz8gbj7aoouiuevmfejwc/diff:/var/lib/docker/overlay2/d38a1c7759c268c8bce5e7c2c7bf4386d94e2a803f93d89e659245c3c6ba8081/diff",
+    "MergedDir": "/var/lib/docker/overlay2/uvuy1xoz7vomuvdzlzrgpu25v/merged",
+    "UpperDir": "/var/lib/docker/overlay2/uvuy1xoz7vomuvdzlzrgpu25v/diff",
+    "WorkDir": "/var/lib/docker/overlay2/uvuy1xoz7vomuvdzlzrgpu25v/work"
+  },
+  "Name": "overlay2"
+}
+```
+
+### Using `dive`
+There is an OSS tool named `dive` that can be used to interactively drill into an image to see the layers, what file 
+changes they have, etc.
+
+```shell
+$ dive custom-node
+Image Source: docker://custom-node
+Fetching image... (this can take a while for large images)
+Analyzing image...
+Building cache...
+```
+
+
+Dive is a curses-based program that can be run from your terminal; in the example below I've selected the layer that we 
+are interested in - the COPY instruction - and can look under the "Layer Details" section to see the Id for that layer.
+
+![Dive01](./images/dive01.png)
+
+#### Backing up the Image
+Now we are going to make use of the `docker save` command to export our image to a tar file
```

Comment:
Assume the reader is unaware of `docker save` and provide an inline link

Reply:
done